modules/swagger-codegen/src/main/resources/typescript-angular/api.mustache

```diff
@@ -39,7 +39,7 @@ namespace {{package}} {
          * {{notes}}
          {{#allParams}}* @param {{paramName}} {{description}}
          {{/allParams}}*/
-        public {{nickname}} ({{#allParams}}{{paramName}}{{^required}}?{{/required}}: {{{dataType}}}, {{/allParams}}extraHttpRequestParams?: any ) : ng.IHttpPromise<{{#returnType}}{{{returnType}}}{{/returnType}}{{^returnType}}{}{{/returnType}}> {
+        public {{nickname}} ({{#allParams}}{{paramName}}{{^required}}?{{/required}}: {{{dataType}}}, {{/allParams}}) : ng.IHttpPromise<{{#returnType}}{{{returnType}}}{{/returnType}}{{^returnType}}{}{{/returnType}}> {
```

Comment:
actually on a second thought, this should only apply to TypeScriptAngular2ClientCodegen